CHANGELOG.md

```diff
@@ -12,39 +19,39 @@ Thanks to the contributions of the community ❤️💙💛💜🧡
 - [dwmcc](https://github.com/dwmcc)
 - [wingyplus](https://github.com/wingyplus)
 
-- Feature: Support OAuth2 for swagger-ui (#217)
-- Feature: Support `default` response type in responses (#301)
-- Feature: Allow overriding `x-struct` in `OpenApiSpex.schema/1` (#304)
-- Feature: Ability to specify `deprecated` in ControllerSpec operation (#296)
-- Feature: `:struct?` and `:derive?` options in `OpenApiSpex.schema/1` (#312)
-- Feature: `OpenApiSpex.add_schemas/2` (#314)
-- Enhancement: Remove api_spec data from Conn (#286)
-- Enhancement: More informative errors for bad schema (#288, #284, #287) 
-- Fix: Convert `:format` value to atom when decoding schema file (#293)
-- Fix: Type spec in OpenApiSpex.Info
-- Fix: Elixir Formatter rules in published package (#306)
-- Docs: Fix spelling error in example code (#295)
-- Docs: Fix type in README (#297)
-- Docs: Fix links and punctuation in README (#298)
-- Docs: Promote ControlerSpecs as the preferred API for controller operations (#311)
-
-# 3.9.0
+* Feature: Support OAuth2 for swagger*ui (#217)
```

Comment:
It looks like a couple of these search-and-replace changes are unintended. For example, `swagger-ui` should not be changed. Can you revert them?

Also, why should markdown list items be demarcated with an asterisk instead of a dash? The dash was used because the Prettier utility put them in.

Reply:
Agreed. Too much churn here.

Reply:
> It looks like a couple of these search-and-replace changes are unintended. For example, `swagger-ui` should not be changed. Can you revert them?

Done.
 
> Also, why should markdown list items be demarcated with an asterisk instead of a dash? The dash was used because the Prettier utility put them in.

The dash was replaced with asterisk because Earmark seemed to render it by adding extra spaces to the bullet list, see the items list difference between between 3.9.0 an 3.8.0 in the screenshot below:

![Screenshot-20210502050214-1280x1440](https://user-images.githubusercontent.com/134518/116794977-0a8a8080-ab04-11eb-8f79-fd5b133391c9.png)